ckanext/pigma_theme/scripts/import.py

```diff
@@ -0,0 +1,105 @@
+    :param path: path to the csv file (local, does not handle URLs)
+    :return: a list of dicts, one dict per group
+    """
+    dataset_dicts = []
+    with open(path, 'rb') as csvfile:
+        reader = csv.reader(csvfile, delimiter=',', quotechar='"')
+        # This skips the first row of the CSV file.
+        next(reader)
+        for row in reader:
+            row_as_dict = {
+                "name": row[0],
+                "title": row[1],
+                "id": row[0],
+                "image_url": row[2]
+            }
+            dataset_dicts.append(row_as_dict)
+    return dataset_dicts
+
+
+def _create_groups(dataset_dicts, API_URI, BASE64AUTH):
+    """
+    Given en list of dicts defining the groups, creates the group using CKAN's API
+    Since geOrchestra, for now, doesn't let in using the API key, we use basic auth.
+    If a group already exists in CKAN, pruges it before creating it again.
+    :param dataset_dicts:
+    :param API_URI: URL to ckan api (e.g.https://georchestra.mydomain.org/ckan/api/3/)
+    :param BASE64AUTH: basic auth chain built with user/passwd
+    :return:
+    """
+    ctx = ssl.create_default_context()
+    ctx.check_hostname = False
+    ctx.verify_mode = ssl.CERT_NONE
+
+    for dataset_dict in dataset_dicts:
+        try:
+            delete_req = urllib2.Request(API_URI + 'action/group_purge')
+            #delete_req.add_header('Authorization', AUTH_TOKEN)
+            delete_req.add_header("Authorization", "Basic %s" % BASE64AUTH)
+            del_response = urllib2.urlopen(delete_req, urllib.quote(json.dumps(dataset_dict)), context=ctx)
+            assert del_response.code == 200
+        except:
+            print('group {} did not exist before'.format(dataset_dict['id']))
+
+        request = urllib2.Request(API_URI + 'action/group_create')
+        #request.add_header('Authorization', AUTH_TOKEN)
+        request.add_header("Authorization", "Basic %s" % BASE64AUTH)
+        response = urllib2.urlopen(request, urllib.quote(json.dumps(dataset_dict)), context=ctx)
+        assert response.code == 200
+
+        response_dict = json.loads(response.read())
+        assert response_dict['success'] is True
+
+        created_package = response_dict['result']
+        pprint_color(created_package)
+
+
+def main():
+    # Input arguments
+    parser = argparse.ArgumentParser(description='''
+    Reads group configuration  in a CSV file dans creates them using CKAN's API. GeOrchestra does not allow API keys, 
+    so we use user/password instead
+    ''')
+    parser.add_argument('ckan_api_url', help='the source name to process')
+    parser.add_argument('-c', '--csv',
+                        help='csv file path')
+    parser.add_argument('-u', '--user',
+                        help='geOrchestra user name')
+    parser.add_argument('-p', '--password',
+                        help='geOrchestra user password')
+    args = parser.parse_args()
+
+    API_URI = args.ckan_api_url
```

Comment:
We don't want to spend much time on this, but maybe adding the possibility to pass api url, username and password via env var could be a good idea ??

Reply:
Instruction from Antoine is keep it simple, don't waste time on it, the aim is to keep a trace so we don't have to write it all over from scratch next time. 
This is targeted as one-shot runs. I would say improvements will be made when necessary.
Anyway, since we use API, it is meant to be run from local machine, probably not on docker.